docs/source/quantization/auto_awq.rst

```diff
@@ -5,21 +5,19 @@ AutoAWQ
 
 .. warning::
 
-   Please note that AWQ support in vLLM is under-optimized at the moment. We would recommend using the unquantized version of the model for better
-   accuracy and higher throughput. Currently, you can use AWQ as a way to reduce memory footprint. As of now, it is more suitable for low latency
-   inference with small number of concurrent requests. vLLM's AWQ implementation have lower throughput than unquantized version.
+   Please note that Marlin kernels are up to 2.59x faster than the AWQ GEMM kernel which is under-optimized in vLLM. You can use the faster kernels by passing setting the `quantization` parameter in vLLM to ``awq_marlin`` instead of ``awq``.
```

Comment:
I don't think you need to specify `awq_marlin`, it will automatically convert AWQ checkpoints if available.

Reply:
Thanks for this comment. I think it may be best to remove the warning altogether if that's the case. Should we let the user know about this in the documentation or will it suffice with a logging message when it is automatically converted?

Reply:
We have a logged message about using the `marlin` kernel. I think that this should be mostly opaque to the user though.

Reply:
Here is the log:
```
pip install https://vllm-wheels.s3.us-west-2.amazonaws.com/42de2cefcbfc1e640f3337b36d5dddfd3133ef0e/vllm-0.5.2-cp38-abi3-manylinux1_x86_64.whl
python
>>> from vllm import LLM 
>>> model = LLM("Sreenington/Phi-3-mini-4k-instruct-AWQ")
INFO 07-22 20:55:05 awq_marlin.py:77] The model is convertible to awq_marlin during runtime. Using awq_marlin kernel.
INFO 07-22 20:55:05 llm_engine.py:175] Initializing an LLM engine (v0.5.2) with config: model='Sreenington/Phi-3-mini-4k-instruct-AWQ', speculative_config=None, tokenizer='Sreenington/Phi-3-mini-4k-instruct-AWQ', skip_tokenizer_init=False, tokenizer_mode=auto, revision=None, rope_scaling=None, rope_theta=None, tokenizer_revision=None, trust_remote_code=False, dtype=torch.float16, max_seq_len=4096, download_dir=None, load_format=LoadFormat.AUTO, tensor_parallel_size=1, pipeline_parallel_size=1, disable_custom_all_reduce=False, quantization=awq_marlin, enforce_eager=False, kv_cache_dtype=auto, quantization_param_path=None, device_config=cuda, decoding_config=DecodingConfig(guided_decoding_backend='outlines'), observability_config=ObservabilityConfig(otlp_traces_endpoint=None), seed=0, served_model_name=Sreenington/Phi-3-mini-4k-instruct-AWQ, use_v2_block_manager=False, enable_prefix_caching=False)
```